bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/messages/ERP1Message.java

```diff
@@ -111,8 +111,12 @@ public ERP1Message(int dataLength, int optionalDataLength, byte[] payload) {
                     teachIn = false;
                     senderId = Arrays.copyOfRange(payload, dataLength - 5, dataLength - 1);
                     break;
-                default:
+                case MSC:
+                    teachIn = false;
+                    senderId = Arrays.copyOfRange(payload, dataLength - 5, dataLength - 1);
```

Comment:
Since it seems to be handled the exact same way as **SIG**, perhaps these two cases could be joined to avoid duplication?